src/users/UserList.js

```diff
@@ -1,13 +1,13 @@
 /* @flow strict-local */
 import React from 'react';
-import { SectionList } from 'react-native';
+import { FlatList } from 'react-native';
```

Comment:
```
[...] all elements will be a static 64px
high.
```

This isn't true yet. 🙂 The elements still have text in them (the user's full name, and possibly their email), the user can change the system font size, and the items expand with the font size increases too much. At the tip of this PR branch, if I add `fontSize: 72` to `componentStyles` in `UserItem`, I can see this:

<img width=400 src="https://user-images.githubusercontent.com/22248748/124663108-a8d5f200-de77-11eb-8e40-ecfa167c62c8.png" />